backend/app/models/pydantic.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional
```

Comment:
Let's change the name of the file, I can imagine `import pydantic` becoming a problem. Feel free to use `schema.py`.